src/plugins/allPlugins.ts

```diff
@@ -49,11 +51,23 @@ export const allPlugins = [
   }),
   makeEvmRpc({
     pluginId: 'amoy',
-    url: 'https://polygon-amoy-bor-rpc.publicnode.com'
+    urls: [
+      'https://api.zan.top/polygon-amoy', // yellow privacy
+      'https://polygon-amoy-public.nodies.app', // yellow privacy
```

Comment:
What's with all these privacy comments?

Reply:
These are from chainlist.org. They're there to verify sorting by privacy first (green).